src/@ionic-native/plugins/zeroconf/index.ts

```diff
@@ -0,0 +1,132 @@
+import { Injectable } from '@angular/core';
+import { Cordova, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+export interface ZeroconfService {
```

Comment:
Yes you're right.

FYI, in cases where we have brand names that need to be capitalized. We capitalize the class name but keep the package name as one word. You can see this in `OneSignal`, `PayPal`, `SQLite` .. etc